test/runtests.jl

```diff
@@ -0,0 +1,77 @@
+using LambdaMaker
+using Test
+
+SAMPLE_FILENAME = "Dockerfile"
+
+@testset "_copy_file" begin
+    RESOURCE_DIR = mkdir("resources")
```

Comment:
Could also use a temp dir to avoid creating dirs and files in the repo (and you don't have to worry about cleaning up):

```
mktempdir() do RESOURCE_DIR
    ...
end
```

Reply:
Oh great idea!